dev/bots/codelabs_build_test.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+readonly SCRIPTS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+readonly ROOT_DIR="$SCRIPTS_DIR/.."
+
+function is_expected_failure() {
+  # A test target was specified with the 'build' command.
+  grep --quiet "is not configured for Running" "$1"
+}
+
+clone_flutter_codelabs="git clone https://github.com/material-components/material-components-flutter-codelabs.git"
+checkout_shrine_complete="git checkout 104-complete"
+log_file="build_log_for_104_complete.txt"
+build_command="flutter build bundle"
+
+# Attempt to build 104-complete Shrine app from the Flutter codelabs
+echo "$clone_flutter_codelabs"
+$clone_flutter_codelabs
+
+cd material-components-flutter-codelabs/mdc_100_series/
+
+echo "$checkout_shrine_complete"
+$checkout_shrine_complete
```

Comment:
instead of echoing then running the command, consider `set -x`. It'll make the script simpler.